taxi_service/settings.py

```diff
@@ -122,7 +122,11 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/4.0/howto/static-files/
 
-STATIC_URL = "static/"
+STATIC_URL = "/static/"
```

Comment:
Code Quality Issue: Similar to the templates directory, it's not a good practice to hardcode the 'static' directory in your STATIC_URL setting. Use the os.path or pathlib module to dynamically get the path to the static directory.